doc: update PROJECT_PROGRESSION.md (#321)
openjs-foundation/cross-project-council: PROJECT_PROGRESSION.md

New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.

#### Roles

##### Applicant

Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.

##### Application Manager

> The application manager ensures progress through the process, helps project understand the work to be completed to move though the

Author: @evenstensberg thanks fixed.

to move though the application process. Typically the application manager will be one of the Foundation staff members.

##### Application Champion

The application champion is a member of the CPC who commits to do the following:

1. Provide weekly update to the applicant on progression through the process.
1. Be available to answer questions from the applicant.
1. Advocate on behalf of the project applying.
1. Help to explain why the project is a good fit.
1. Help the application manager when needed.

#### Process

1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).

> 1. An acknowledgement is sent to the application by Foundation staff within 48 hours (striving to reply within 24hours when possible).

Reviewer: ```suggestion
1. An acknowledgement is sent to the applicant by Foundation staff within 48 hours (striving to reply within 24 hours when possible).
```

1. Silent period. Internal only discussion.
1. Initial acceptance as [Incubating][] Project. Before public announcement there will be:

>    * Go/no go confirmation to the applicant that they are ready to have their application become public.

Reviewer: I'm confused about the process here.

From the perspective of an applicant, what would be ideal is to get the below mail sent and collect all of the CPC member approval, and then get back to the applicant with the go-ahead to sync on an announcement date.

The sync bit is particularly important for larger projects.

Author: Flipped the order and added a line about agreeing on an announce date.

   * Private email to the CPC members which asks for +1 or objections to the application becoming public. This email will include:
     * The name of the application manager for the application

>      * The name of the application chamption for the application

Reviewer: ```suggestion
     * The name of the application champion for the application
```

     * Completed application 

>      * Completed application fit checklist.

Reviewer: ```suggestion
     * Completed application-fit checklist.
```

     * Confirmation that the board has had an FYI and there were no objections.
     * Confirmation that there are no existing objections from CPC members.
     This is not a formal vote just the process to ensure we have regular CPC concensus to move forward.
1. Project goes through process of adhering to [on-boarding checklist](#onboarding-checklist).
1. Project presents to CPC for final acceptance via 2/3 supermajority vote and appropriate initial stage is determined.

> ##### Initial fit checklist

Reviewer: I'm curious when/how you're planning to gather some of this data. I'm filling in an application right now.  Although my answers to some of these would be obvious and quite clear were they asked, I don't really see how you'll get that info from the application itself.

Reviewer: In particular, I'm thinking about:

1. Is project open/willing to move to more open governance?
1. Will project adopt the OpenJS CoC?

Author: I think there are questions in the application about the current governance and the existing CoC but we may need to adjust them to ask those 2 questions.

Author: Until then, we'd ask that as part of the discussion in the quiet period.

Author: When ahead an added update to NEW_PROJECT_APPLICATION.md to ask those questions specifically.

1. Is project open/willing to move to more open governance?
1. Will project adopt the OpenJS CoC?
1. Licences are compatible with Foundation requirements.

> 1. No red flags from website, social media accounts etc.

Reviewer: [](url)
```suggestion
1. No red flags from website, social media accounts, etc.
```

> 1. Fit with OpenJS Foundation (not all of these need to met, but provide different ways of sessing strategic fit)

Reviewer: ```suggestion
1. Fit with OpenJS Foundation (not all of these need to met, but provide different ways of assessing strategic fit).
```

New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.

#### Roles

##### Applicant

> Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.

Reviewer 2: How do we verify that an applicant can represent the project in question?

Reviewer 3: Do we need to verify this? Isn't it simply another form of a Developer Certificate of Origin (DCO)?

Author: We have that later in the fit checklist. We purposely left if vague as to how we'd validate that.

##### Application Manager

The application manager ensures progress through the process, helps the project understand the work to be completed and
to move though the application process. Typically the application manager will be one of the Foundation staff members.

##### Application Champion

The application champion is a member of the CPC who commits to do the following:

1. Provide weekly update to the applicant on progression through the process.
1. Be available to answer questions from the applicant.
1. Advocate on behalf of the project applying.
1. Help to explain why the project is a good fit.
1. Help the application manager when needed.

#### Process

1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
1. An acknowledgement is sent to the applicant by Foundation staff within 48 hours (striving to reply within 24 hours when possible).
1. Silent period. Internal only discussion.

> 1. Initial acceptance as [Incubating][] Project. Before public announcement there will be:

Reviewer 2: ```suggestion
1. Initial acceptance as an [Incubating][] Project. Before public announcement there will be:
```

   * Private email to the CPC members which asks for +1 or objections to the application becoming public. This email will include:
     * The name of the application manager for the application
     * The name of the application champion for the application
     * Completed application 
     * Completed application-fit checklist.
     * Confirmation that the board has had an FYI and there were no objections.
     * Confirmation that there are no existing objections from CPC members.
     This is not a formal vote just the process to ensure we have regular CPC concensus to move forward.

>    * Go/no go confirmation to the applicant that they are ready to have their application become public.

Reviewer 4: I would suggest breaking this bit into two paths
```
 If there is agreement to move forward:
   * Confirmation to the applicant that they are ready to have their application become public.
   * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date. Confirm next steps beyond announcement.
If lack of consensus to move forward:
    * Correspondence from foundation staff to applicant about non-acceptance.
    * If possible, give guidance to the project in their quest to find a foundation home (ex. perhaps a different foundation would be a more suitable endeavor.)
```

Author: done

   * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
1. Project goes through process of adhering to [on-boarding checklist](#onboarding-checklist).
1. Project presents to CPC for final acceptance via 2/3 supermajority vote and appropriate initial stage is determined.

##### Initial fit checklist

> 1. Is project open/willing to move to more open governance?

Reviewer 4: As I've stated above in review, I take issue with the half-measure of "more open governance." I suggest we remove the word "more."

Reviewer 5: Some applicants (specially those of small projects) might not now what "open governance" means. Maybe we should link to some examples?

1. Will project adopt the OpenJS CoC?
1. Licences are compatible with Foundation requirements.
1. No red flags from website, social media accounts, etc.

> 1. Fit with OpenJS Foundation (not all of these need to met, but provide different ways of assessing strategic fit).

Reviewer 4: "need to med" should be "need to be met"

     * Confirmation that there are no existing objections from CPC members.
     This is not a formal vote just the process to ensure we have regular CPC concensus to move forward.
   * If there is agreement to move forward:
     * Confirmation to the applicant that they are ready to have their application become public.
     * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
       Confirm next steps beyond announcement.  
   * If lack of consensus to move forward:
      * Correspondence from foundation staff to applicant about non-acceptance.
      * If possible, give guidance to the project in their quest to find a foundation home
        (ex. perhaps a different foundation would be a more suitable endeavor.)
   * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
1. Project goes through process of adhering to [on-boarding checklist](#onboarding-checklist).
1. Project presents to CPC for final acceptance via 2/3 supermajority vote and appropriate initial stage is determined.

##### Initial fit checklist

> 

Reviewer 6: We should likely include something below about initial review of items to be legally transferred

e.g. 

* What Trade Marks will be transferred
* What domains will be transferred
* What IP currently exists and how was it licensed. ( this one is similar to existing point about license being compatible)

Author: @MylesBorins , do those 3 need to be captured/resolved during the quiet period?  If so we should add to the list, if not then maybe better in a checklist for the final vote?

Reviewer 6: I think this is something to check with board / legal. We likely want to do a very high level review to ensure their are no obvious conflicts

Author: @MylesBorins added those three, and we can ask board about legal review in the next meeting

Author: We discussed in board meeting, I think the answer was no.  @MylesBorins FYI.

Author: We discussed in the recent board meeting and I think the answer was that we don't need a high-level legal review during quiet period.  Any required review will be handled during the incubation phase.

New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.

#### Roles

##### Applicant

Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.

##### Application Manager

The application manager ensures progress through the process, helps the project understand the work to be completed and

> to move though the application process. Typically the application manager will be one of the Foundation staff members.

Reviewer 3: I would lean into the language here a little more. "Typically" is ambiguous as to who _else_ is a valid Application Manager. In an attempt to be explicit I've suggested two other sets of folks to consider. Please feel free to accept that portion of the suggestion or not.

```md
The Application Manager ensures progress through the process, helps the project understand the work to be 
completed along with explaining where they are in the application process. Applications Managers can be a
member of the OpenJS Foundation staff, CPC Member, or other person approved by the CPC.
```

##### Application Champion

> The application champion is a member of the CPC who commits to do the following:

Reviewer 3: Consistent casing:

```suggestion
The Application Champion is a member of the CPC who commits to do the following:
```

1. Provide weekly update to the applicant on progression through the process.
1. Be available to answer questions from the applicant.
1. Advocate on behalf of the project applying.
1. Help to explain why the project is a good fit.

> 1. Help the application manager when needed.

Reviewer 3: Moar casing nits. Feel free to ignore them if I'm being too detailed:

``` md
1. Provide weekly update to the Applicant on progression through the process.
1. Be available to answer questions from the Applicant.
1. Advocate on behalf of the Project applying.
1. Help to explain why the Project is a good fit.
1. Help the Application Manager when needed.
```

#### Process

1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
1. An acknowledgement is sent to the applicant by Foundation staff within 48 hours (striving to reply within 24 hours when possible).

> 1. Silent period. Internal only discussion.

Reviewer: ```suggestion
1. Silent period. Internal only discussion. The content of these discussion along with the content of the application itself are kept confidential.
```

Closes #327.

> 1. Initial acceptance as an [Incubating][] Project. Before public announcement there will be:

Reviewer 3: Nitpick: the `[]` after `[Incubating]` is extraneous. This is valid markdown:

```suggestion
1. Initial acceptance as an [Incubating] Project. Before public announcement there will be:
```

New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.

#### Roles

##### Applicant

Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.

##### Application Manager

> he Application Manager ensures progress through the process, helps the project understand the work to be 

Reviewer: ```suggestion
The Application Manager ensures progress through the process, helps the project understand the work to be 
```

completed along with explaining where they are in the application process. Applications Managers can be a
member of the OpenJS Foundation staff, CPC Member, or other person approved by the CPC.

##### Application Champion

> The Application Champion is a member of the CPC who commits to do the following:

Reviewer 7: Can a member of the CPC be the same person sending the application?

Author: I don't think we've discussed one way or the other. My first thought is that there might be some conflict of interest but I'm not sure.  On the other hand it would make some aspects easier. I think we should discuss this outside of this PR which is going to land soon.  @mcollina can you open a new issue to discuss?

Reviewer 7: Sure!
